src/tiled/newtilesetdialog.cpp

```diff
@@ -284,6 +282,11 @@ void NewTilesetDialog::updateOkButton()
  */
 void NewTilesetDialog::pickColorFromImage()
 {
+    if (mPopup != nullptr) {
```

Comment:
I think you didn't test this, because this check is the wrong way around. Also, it should have been written shorter as `if (!mPopup) {`.